webpack/envs/baseConfig.js

```diff
@@ -102,6 +102,21 @@ const baseConfig = {
   externals: {
     request: 'request',
   },
+  optimization: {
+    runtimeChunk: {
+      name: 'runtime',
+    },
+    splitChunks: {
+      cacheGroups: {
+        runtimeChunk: 'single',
+        vendor: {
+          name: 'vendor',
+          chunks: 'initial',
+          test: /[\\/]node_modules[\\/]/,
+        },
+      },
+    },
+  },
```

Comment:
Before making this change, try updating the base webpack config with these two settings, under `resolve.alias`: 

https://github.com/artsy/force/blob/master/webpack/envs/baseConfig.js#L81-L82

This ensures that things resolve to a single instance. The react one is also important because a similar problem happens with hooks. (The _underlying_ problem is that with React 16.3's new context api, context usage in React and Styled Components is now static, derived from a top level export, rather than an instance, derived from `new`.)

Reply:
Not relevant to this PR but good to know if you ever plan on breaking out the UI system into it's own package and want to use `yarn link` in combination with SSR. This is what fixed it for us: 

https://github.com/damassi/force/blob/7b13d63872d4d5748029def19f5dcd9ae25f0841/src/index.js#L31-L42

Reply:
Much thanks!